test/entry/flag-entry/entry-with-flag.test.js

```diff
@@ -39,8 +49,8 @@ describe('entry flag', () => {
         });
     });
 
-    it('should resolve the path to src/a.js as ./src/a.js', (done) => {
-        const { stderr, stdout } = run(__dirname, ['--entry', 'src/a.js']);
+    it('should resolve the path to /src/a.js as ./src/a.js', (done) => {
+        const { stderr, stdout } = run(__dirname, ['--entry', '/src/a.js']);
```

Comment:
webpack 5 doesn't resolve `src/a.js` as `./src/a.js` but this test was passing earlier because of the old output file created above.

but it resolves `/src/a.js` as `./src/ajs`.

Reply:
ah I see, seems valid then, let's do it at top level then? Keep a list of test dirs where the files are to be removed and run it for that list

Reply:
for each test

Reply:
Can you explain more?

Reply:
can we use global beforeEach for certain tests

Reply:
I am not sure if that is possible. Also, all beforeEach statements aren't exactly the same.